waiter/test/waiter/core_test.clj

```diff
@@ -1108,7 +1108,8 @@
       (is (not (waiter-request?-fn {:headers {"host" "waiter-host-1"}})))))
 
   (testing "list hostname config"
-    (let [config {:settings {:hostname ["waiter-host-1" "waiter-host-2"]}}
+    (let [config {:settings {:hostname "waiter-host-1"
+                             :alternate-hostnames ["waiter-host-2"]}}
```

Comment:
Lexicographic order :(

Reply:
This is a case where I believe it's reasonable to have `:hostname` first.

Reply:
Same here